app/presenters/chf_index_presenter.rb

```diff
@@ -0,0 +1,26 @@
+class ChfIndexPresenter < Blacklight::IndexPresenter
+  class_attribute :description_max_chars
+  self.description_max_chars = 10
+
+  attr_reader :description_field
+
+  def initialize(*args)
+    super
+    # This is an attempt to not hard-code in that our field is called
+    # `description_tesim`, so that doesn't break if it's changed, we
+    # just look for the "description" field. Not really sure if this is
+    # really less fragile, or if there was a better place to get this.
+    @description_field = configuration.index_fields.values.
+      find { |conf| conf.itemprop == "description" }.
```

Comment:
Yeah that's not what I meant. I guess; where is this configuration? where do i go to look at index_fields? I want to take a look.